.github/workflows/intel.yml

```diff
@@ -24,8 +24,12 @@ jobs:
       CXX: icpx
       DEBIAN_FRONTEND: noninteractive
     steps:
+      - name: Setup cmake
+        uses: jwlawson/actions-setup-cmake@v2
+        with:
+          cmake-version: '3.30.x'
       - name: Install required packages to install Intel compilers and to build
-        run: apt -y update && apt install -y bzip2 cmake git gnupg ninja-build wget
+        run: apt -y update && apt install -y bzip2 git gnupg ninja-build make wget
```

Comment:
Why do you need ninja and make? Should be fine using ninja generator.

Reply:
The build itself went fine, I believe it was the demos build step in the CI that failed if `make` is not available.

Reply:
I guess cmake installs make which isn't technically correct.

You can leave it as it is, good to have some test coverage using make.